lib/ui/compositing.dart

```diff
@@ -143,16 +146,19 @@ class SceneBuilder extends NativeFieldWrapperClass2 {
   /// Pushes a physical layer operation for an arbitrary shape onto the
   /// operation stack.
   ///
-  /// Rasterization will be clipped to the given shape defined by [path]. If
-  /// [elevation] is greater than 0.0, then a shadow is drawn around the layer.
-  /// [shadowColor] defines the color of the shadow if present and [color] defines the 
+  /// By default, the layer's content will not be clipped (clip = [Clip.none]).
+  /// If clipMode is nonzero (Clip.hardEdge, Clip.antiAlias, or Clip.antiAliasWithSaveLayer),
```

Comment:
please put those values in square brackets

also, rather than "is nonzero" consider "has another value" since the enum values aren't zero.

Reply:
Thanks! I clearly forgot to update the comment here when I changed `int clipMode` to `Clip clip`. Updating now.